pkg/api/dtos/user.go

```diff
@@ -14,10 +14,11 @@ type SignUpStep2Form struct {
 }
 
 type AdminCreateUserForm struct {
-	Email    string `json:"email"`
-	Login    string `json:"login"`
-	Name     string `json:"name"`
-	Password string `json:"password" binding:"Required"`
+	Email         string `json:"email"`
+	Login         string `json:"login"`
+	Name          string `json:"name"`
+	Password      string `json:"password"`
+	ThirdPartUser bool   `json:"is_thirdpart"`
```

Comment:
Would not ShouldGenPassword be more descriptive. Oh, and underscores in json is no go

Reply:
Yeah, `ShouldGenPassword` is better.